### DISCUSSION TITLE
UI/Theme Changes

### DISCUSSION THREAD

**Discussion by legendword** [legendword/blog#19] [General]
Proposing changes to the color theme and various UI layouts:

- **Consistency between pages**. Achieved through breaking pages down into more individual components.
- **Multi-tone color theme**. Deciding on a secondary color.
- **Changing primary color**. Needs to change logo color accordingly.
- **Flat vs. shadow**. Flat cards/headers may look better.

Further input is appreciated!
@nekopudding @Lisherex

**Comment by nekopudding** [comment]
- __More consistent style__. It is a good idea to fix inconsistencies such as between the style of the creater's profile page and the collections page. Making the layout more consistent will also give the users an easier time navigating the website.
  - __heading size consistency.__ The heading sizes and color are inconsistent across Discovery, Collections, and Following
- __Line spacings and headings can be readjusted__. The discover page has too many large headings that compete with each other. Either reduce the "Categories" font size or get rid of it. It also looks weird that such a short section has the same big heading as "All Posts", and there isn't a clear division between the two sections. We could try framing the categories section with a different background or use a divider for separating the sections.
- The use of bright colors for the categories section as opposed to the dull color for the posts seem to bring too much emphasis on the categories themselves as opposed to the actual content which are the posts, could try hollowing out the category background, or changing the background color of the website.
- There are inconsistencies in the font weight when typing in mandarin with numbers. For example, in a post with a title "符文工房5真垃圾", the "5" seems to be bolded which stands out from the rest of the characters. I do not know a fix for this.
- The beta testing banner box edges seems to be misaligned with the rest of the content.
- __Shadow vs flat__ - agree that a flat card style fits better with the theme, the shadows, especially on the posts grid look a bit agressive. though it may be a good idea to find another way to bring the cards out from the background without the use of shadows.
- __Primary color__ - logo would match better if its color matches the main color used in the website - right now the logo's color doesn't seem be anywhere obvious on the website.
- __Compose page__.  The Title and Description zooming animation seem a bit unecessary and distracting. The Preview heading is misaligned. It seems that the current intention for this page is to have the authors compose their posts directly on this page. If there isn't one already, it may be a good idea to add a feature to save drafts, spelling checking. Right now, it's hard to focus having to view Preview and the Input Field side by side simultaneously all the time. It may be a good idea to just move the preview section downward or into a toggle, and increase the horizontal padding from the edge of the browser so its less awkward for the user's eyes to have to read text near the edge of the screen (awkward when you fullscreen the browser). Using reddit as reference, I suggest keeping the input field fairly centered and no wider than half of the laptop screen. Find useful things to fill in the empty space on page, like quick navigation links or useful tools for composition.
